content/docs/cli.md

```diff
@@ -138,7 +138,20 @@ A specific application version on the official GitHub repo. (default to the most
 :::
 
 ::: flag -p, --parameter (stringArray)
-The parameter name and value separated by '='
+The parameter name and value separated by '='. See also `-P`
+:::
+
+::: flag -P, --parameter-file (stringArray)
+Path to a YAML file with parameter values. The top-level element in this file must be a mapping,
+where keys are parameter names and values are the parameter values.
```

Comment:
I suggest an example mapping here

Reply:
I added a pointer to the one below.

Reply:
Wdyt about a more complex example with nested parameter values?

Reply:
I'd say let's have an actual operator which can consume such parameters and then we can use it as a non-vaporware example.